File: analyze-posts.js
const fs = require('fs');
const path = '/Users/zazouino/Downloads/voglansastro-main/src/data/posts.js';
const content = fs.readFileSync(path, 'utf-8');

// Extraire les catégories
const categories = {};
const lines = content.split('\n');
lines.forEach(line => {
  const match = line.match(/category:\s*"([^"]+)"/);
  if (match) {
    const cat = match[1];
    categories[cat] = (categories[cat] || 0) + 1;
  }
});

console.log('📊 ANALYSE DES 300 ARTICLES\n');
console.log('RÉPARTITION PAR CATÉGORIE:');
Object.entries(categories)
  .sort((a, b) => b[1] - a[1])
  .forEach(([cat, count]) => {
    console.log(`  ${cat}: ${count} articles`);
  });

console.log(`\nTOTAL: ${Object.values(categories).reduce((a,b)=>a+b,0)} articles`);

File: analyze-posts.py
#!/usr/bin/env python3
import re

with open('src/data/posts.js', 'r') as f:
    content = f.read()

# Compter les articles
articles = re.findall(r'href:\s*"([^"]+)"', content)
categories = re.findall(r'category:\s*"([^"]+)"', content)

print(f"📊 ANALYSE DES 300 ARTICLES DU BLOG\n")
print(f"✅ Total articles trouvés: {len(articles)}\n")

# Compter par catégorie
from collections import Counter
cat_counts = Counter(categories)

print("RÉPARTITION PAR CATÉGORIE:")
for cat, count in cat_counts.most_common():
    print(f"  • {cat}: {count} articles")

print(f"\n📝 TOTAL VÉRIFIÉ: {sum(cat_counts.values())} articles")
